scripts/src/main/resources/variables.bat

```diff
@@ -13,18 +13,14 @@ set MAIN_BRANCH=main
 set SCRIPTS_PATH=scripts
 set CONF_PATH=conf
 set WORKSPACES_PATH=workspaces
+set WORKSPACESVS_PATH=workspaces_vs
 rem set SETTINGS_REL_PATH=settings
 set SETTINGS_REL_PATH=development\settings
+set ECLIPSE_TEMPLATES_REL_PATH=eclipse\workspace
 rem absolute software path
-if "%CD:~-1%" == "/" (
-  set SOFTWARE_PATH=%CD%software
-) else (
-  if "%CD:~-1%" == "\" (
-    set SOFTWARE_PATH=%CD%software
-  ) else (
-    set SOFTWARE_PATH=%CD%/software
-  )
-)
```

Comment:
This IFs have been written on purpose. If there is a duplicate slash then windows causes some errors.